packages/react-components/src/AccessibilityScenarios/Link.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Link } from '@fluentui/react-link';
+
+import { Scenario } from './utils';
+
+export const SiteNavigationAccessibilityScenario: React.FunctionComponent = () => {
+  return (
+    <Scenario pageTitle="Site navigation links">
+      <ul>
+        <li>
+          <Link href="https://www.microsoft.com" target="_blank">
+            Microsoft
+          </Link>
+        </li>
+        <li>
+          <Link href="https://www.office.com" target="_blank">
+            Microsoft Office
+          </Link>
+        </li>
+        <li>
+          <Link href="https://www.github.com" target="_blank">
+            GitHub
+          </Link>
+        </li>
+        <li>
+          <Link href="https://www.linkedin.com" target="_blank">
+            LinkedIn
+          </Link>
+        </li>
+        <li>
+          <Link href="https://www.skype.com" target="_blank" disabled>
+            Link can be disabled
+          </Link>
+        </li>
+        <li>
+          <Link href="https://www.bing.com" target="_blank" disabledFocusable>
+            Link can be disabled but focusable
```

Comment:
```suggestion
            Bing
```
Same as above, for user testing (and just in general) it seems better to have link text reflect the link purpose, not the role and state.